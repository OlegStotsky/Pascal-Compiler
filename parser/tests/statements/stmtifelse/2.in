var
	a : integer;
	b : integer;

begin
	if a < b then
	begin
		a := 4;
		a := 3;
	end;
end;