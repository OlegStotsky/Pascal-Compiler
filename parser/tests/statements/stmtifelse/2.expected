BEGIN SYMTABLE
 VAR
  NAME : a
  TYPE : integer
 VAR
  NAME : b
  TYPE : integer
 TYPE : boolean
 TYPE : char
 TYPE : integer
 TYPE : float
END SYMTABLE
BEGIN STMT BLOCK
 BEGIN STMT IF-ELSE
  IF
    a
   <
    b
  THEN
   BEGIN STMT BLOCK
    BEGIN STMT ASSIGN
      a
     :=
      4
    END STMT ASSIGN
    BEGIN STMT ASSIGN
      a
     :=
      3
    END STMT ASSIGN
   END STMT BLOCK
 END STMT IF-ELSE
END STMT BLOCK
